modules/cdc-ext/src/test/java/org/apache/ignite/cdc/kafka/CdcKafkaReplicationTest.java

```diff
@@ -139,11 +155,85 @@ public class CdcKafkaReplicationTest extends AbstractReplicationTest {
         return futs;
     }
 
+    /** {@inheritDoc} */
+    @Override protected void checkMetrics() throws IgniteInterruptedCheckedException {
+        super.checkMetrics();
+
+        for (AbstractKafkaToIgniteCdcStreamer k2i : k2is) {
+            KafkaToIgniteMetrics metrics = getFieldValue(k2i, "metrics");
+            MetricRegistryImpl mreg = getFieldValue(metrics, "mreg");
+            checkK2IMetrics(m -> mreg.<AtomicLongMetric>findMetric(m).value());
+        }
+    }
+
     /** {@inheritDoc} */
     @Override protected void checkConsumerMetrics(Function<String, Long> longMetric) {
-        assertNotNull(longMetric.apply(IgniteToIgniteCdcStreamer.LAST_EVT_TIME));
-        assertNotNull(longMetric.apply(IgniteToIgniteCdcStreamer.EVTS_CNT));
-        assertNotNull(longMetric.apply(IgniteToKafkaCdcStreamer.BYTES_SENT));
+        assertNotNull(longMetric.apply(LAST_EVT_SNT_TIME));
+        assertNotNull(longMetric.apply(EVTS_SNT_CNT));
+        assertNotNull(longMetric.apply(BYTES_SNT));
+    }
+
+    /** {@inheritDoc} */
+    @Override protected void checkMetricsCount(int putCnt, int rmvCnt) {
```

Comment:
Looks like we can union `putCnt` and `rmvCnt` to `evtsCnt` (and I think we can pass `KEYS_CNT` and `KEYS_CNT * 2` here)

Reply:
Yeah, I agree. I refactored this part of the tests